Makefile

```diff
@@ -35,6 +35,11 @@ RPM_EXTRAFLAGS := \
 
 IMAGE_REGISTRY :=registry.svc.ci.openshift.org
 
+oc: GO_BUILD_PACKAGES:=./cmd/oc
```

Comment:
I like the idea, although I wonder if we want to introduce such granular targets. None of the other repos have it, I don't want oc to be the raisin :wink: 
/hold

Reply:
I think the one most useful target could be justified, I don't feel strong